### DISCUSSION TITLE
RAS Uncorrectable Error in IOB, base=0xe010000

### DISCUSSION THREAD

**Discussion by albertbu** [OE4T/meta-tegra#1624] [General]
Hi, 
while burning a build from oe_tegra/scarthgap , we get this nasty message
```
 ��ERROR:   Exception reason=0 syndrome=0xbe000011
ERROR:   **************************************
ERROR:   RAS Uncorrectable Error in IOB, base=0xe010000:
ERROR:          Status = 0xec000612
ERROR:   SERR = Error response from slave: 0x12
ERROR:          IERR = CBB Interface Error: 0x6
ERROR:          Overflow (there may be more errors) - Uncorrectable
ERROR:          MISC0 = 0xc4560040
ERROR:          MISC1 = 0x2a4c860000000000
ERROR:          MISC2 = 0x0
ERROR:          MISC3 = 0x0
ERROR:          ADDR = 0x8000000003f300c0
ERROR:   **************************************
ERROR:   sdei_dispatch_event returned -1
ERROR:   **************************************
ERROR:   RAS Uncorrectable Error in ACI, base=0xe01a000:
ERROR:          Status = 0xe8000904
ERROR:   SERR = Assertion failure: 0x4
ERROR:          IERR = FillWrite Error: 0x9
ERROR:          Overflow (there may be more errors) - Uncorrectable
ERROR:          ADDR = 0x8000000003f300c0
ERROR:   **************************************
ERROR:   sdei_dispatch_event returned -1
ERROR:   Powering off core
```
and naturally  the burning process halts. 
I need to say that this is the first time we encounter this, and IU saw this issue occurred in the past on Jetson Pack 35.5
If I am not mistaken, the file mentioned there does not even exist in the current Linux distro 
Thanks in advance

**Comment by kekiefer** [comment]
For what it's worth, RAS indicates an error accessing 0x3f300c0, which is in the address space for the control of one of the p2u phys (p2u_gbe_1 on Orin). Do you have any modifications or custom hardware? This issue on the NVIDIA forums has the same signature: https://forums.developer.nvidia.com/t/orin-nx-8gb-c9-can-not-get-pcie-c7-c9/249948

**Reply by albertbu** [reply]
HI @kekiefer, thanks
We've tried to play according to the link you sent us. We got no improvements. Now we are investigating the pinmux in our system, maybe it's not correct. We are also facing an issue that the recovery boot hangs. Could you pls have a look at the question I asked a few minutes ago here? Thanks
